README.md

```diff
@@ -21,6 +23,16 @@ For FIPS module, you need to generate hmac checksum:
 
     $ ./hmacify.sh target/release/libkryoptic_pkcs11.so
 
+The default build specifies "standard" as the default feature for
+ease of use. "Standard" pulls in all the standard algorithms and the
+sqlitedb storage backedn.
+
+In order to make a different selection you need to use the cargo
+switch to disable default features (--no-default-features) and then
+specify the features you want to build with:
+
+eg: cargo build --no-default-features --features fips,sqlitedb,nssdb
```

Comment:
this example line does not match what is execuded in CI nor in the Makefile, where the `rusqlite/bundled` is used too. I see you removed it from the `fips` feature but it is not clear to me why it can not stay under the "fips" feature to make the feature selection easier for the user.

Reply:
the reason I had removed it is that in theory you could have fips with a non-sqlite db, but currently we support only sqlite or nssdb ... so ... I'll put it back

Reply:
My point was that make has the following line:
```
cargo build --no-default-features --features fips,rusqlite/bundled,sqlitedb,nssdb
```
but the readme this one:
```
cargo build --no-default-features --features fips,sqlitedb,nssdb
```
I did not test either of them, but either one is needlessly complicated or the other is missing something.

Reply:
Yeah, I forgot to change the Makefile and tests with the last push :)